docs/splitting_up.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Splitting up your configuration
+nav_order: 1
+parent: Configuration
+---
+
+# Splitting up your configuration
+
+## Basic usage
+
+XKNX is controlled via a configuration file. Per default the configuration file is named `xknx.yaml`.
```

Comment:
- maybe add a note where this file is looked for (directory) and how to configure its path/name.
- there are a lot of line breaks in sentences in this .md - I guess this could render awkward on some displays.

Reply:
fixed